tests/mysqli_azure_redirection_on.phpt

```diff
@@ -0,0 +1,68 @@
+--TEST--
+Azure redirection test for servers when mysqlnd_azure.enableRedirect="on"
+--INI--
+mysqlnd_azure.enableRedirect="on"
+--SKIPIF--
+<?php
+require_once('skipif.inc');
+?>
+--FILE--
+<?php
+require_once("connect.inc");
+
+//Step 1: check option setting
+$tmp = ini_get("mysqlnd_azure.enableRedirect");
+echo $tmp."\n";
+if (strcmp($tmp, "on")!=0)
+{
+	printf("[001] mysqlnd_azure.enableRedirect not set to on, got '%s'\n", $tmp);
+    die();
+}
+
+//Step 2: check connection result when use SSL
+$link = mysqli_init();
+$ret = @mysqli_real_connect($link, $host, $user, $passwd, $db, $port, NULL, MYSQLI_CLIENT_SSL);
+
+$last_message = "";
+if($ret)
+    $last_message =  $link->info;
+
+//Server supports redirection
+if(strlen($last_message) > 27 && strcmp(substr($last_message, 0, strlen("Location:")), "Location:")==0) {
+    if(substr($link->host_info, 0, strlen($host)) == $host)
+        echo "[003] fail\n";
+    else
+        echo "[003] pass\n";
+}
+else { //Server does not support redirection
+    $lastError = error_get_last()["message"];
+    if (strpos($lastError, "Abort the connection because MySQL server does not enable redirection or network package doesn't meet redirection protocol.") !== false)
```

Comment:
Can we update this error message to: "Connection aborted because redirection is not enabled on the MySQL server or the network package doesn't meet meet redirection protocol."

Reply:
Has updated the message.